Zend/zend_API.h

```diff
@@ -636,6 +636,20 @@ END_EXTERN_C()
 		ZVAL_PSTRINGL(z, "", 0);				\
 	} while (0)
 
+#define ZVAL_ONE_CHAR_STRING(z, c)  do {		\
+		char _c = (c);                          \
+		ZVAL_INTERNED_STR(z, ZSTR_CHAR((zend_uchar) _c));	\
+	} while (0)
+
+#define ZVAL_STRINGL_FAST(z, s, l) do {			\
```

Comment:
Can you please add a comment to clarify when the fast implementation should be used? I believe there are some people (including me) who would appreciate a little bit more information about its applicability/use-cases.

Reply:
@kocsismate 
sorry for my poor English...😢
you can add comments if possible...